src/purchases.ts

```diff
@@ -402,15 +416,15 @@ export default class Purchases {
   /**
    * Fetch the product info
    * @param {String[]} productIdentifiers Array of product identifiers
-   * @param {String} type Optional type of products to fetch, can be inapp or subs. Subs by default
+   * @param {String} type Optional type of products to fetch, can be SUBSCRIPTION or NON_SUBSCRIPTION. SUBSCRIPTION by default
    * @returns {Promise<PurchasesStoreProduct[]>} A promise containing an array of products. The promise will be rejected
    * if the products are not properly configured in RevenueCat or if there is another error retrieving them.
    * Rejections return an error code, and a userInfo object with more information. The promise will also be rejected
    * if configure has not been called yet.
    */
   public static async getProducts(
     productIdentifiers: string[],
-    type: PURCHASE_TYPE = PURCHASE_TYPE.SUBS
+    type: PURCHASE_TYPE | PRODUCT_CATEGORY = PRODUCT_CATEGORY.SUBSCRIPTION
```

Comment:
I know it was a pain to keep this function accepting `PURCHASE_TYPE`, but thank you for going the extra mile there. I think it's worth it

Reply:
Less migration notes if we kept it 😛